MAINTAINERS.md

```diff
@@ -4,6 +4,10 @@
 | Saurabh Singh          | [getsaurabh02](https://github.com/getsaurabh02)          | Amazon |
 | Subhobrata Dey         | [sbcd90](https://github.com/sbcd90)                      | Amazon |
 | Praveen Sameneni       | [praveensameneni](https://github.com/praveensameneni)    | Amazon |
```

Comment:
Seems all maintainers should exist in codeowners file.

Reply:
Thanks and done